src/server/game/Miscellaneous/Language.h

```diff
@@ -1157,8 +1157,9 @@ enum TrinityStrings
     LANG_COMMAND_SPY_LIST                = 5069,
     LANG_COMMAND_SPY_WHISPERS_PLAYER     = 5070,
     LANG_COMMAND_SPY_TELLS_GROUP         = 5071,
+    LANG_COMMAND_SPY_SELF                = 5072,
 
-    // Room for more Trinity strings      5072-9999
+    // Room for more Trinity strings      5073-9999
```

Comment:
I know it's not you who wrote this but "Room for ..." sounds weird to me, is it correct english? If someone is not well versed into english, he might not understand what this room is for, no? xD

Reply:
Don't blame me, that's in the original file, so I'm not gonna change it

Reply:
> I know it's not you who wrote this